hpc_provisioner/src/hpc_provisioner/aws_queries.py

```diff
@@ -37,18 +46,50 @@ class CouldNotDetermineKeyPairException(Exception):
     """Indicates that we either found too many or no keypairs with tags HPC_Goal:compute_cluster"""
```

Comment:
This exception class can now be removed?

Reply:
Good catch!